pom.xml

```diff
@@ -1092,26 +1055,14 @@
                   <property name="format" value="org[.]apache[.]commons[.]math[.]" />
                   <property name="message" value="Use commons-math3 (org.apache.commons.math3.*)" />
                 </module>
-                <module name="RegexpSinglelineJava">
-                  <property name="format" value="junit[.]framework[.]TestCase" />
-                  <property name="message" value="Use JUnit4+ @Test annotation instead of TestCase" />
-                </module>
```

Comment:
I don't think this line should have been removed. This was checking to make sure we were *not* using JUnit 3. Instead of removing this, we should have added the checkstyle to make sure we were *not* using JUnit 3 **or** 4. Then, we could get rid of the junit script that was being run during GitHub Actions, as checkstyle would have covered it.